upload-apk-s3.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+path=$1
+file=$2
+AWS_ACCESS_KEY_ID=$3
```

Comment:
Do we really need this as parameters? We could use the environment variables themselves.